src/release/rollups.md

```diff
@@ -38,32 +38,34 @@ queue has been merged.
 2. Run the following command in the pull request thread:
 
     ```
-    @bors r+ rollup=never p=<NUMBER_OF_PRS_IN_ROLLUP>
+    @bors r+ rollup=never p=5
     ````
 
 3. If the rollup fails, use the logs rust-log-analyzer
    provides to bisect the failure to a specific PR and do
    `@bors r-`. If the PR is running, you need to do `@bors r- retry`. Otherwise,
    your rollup succeeded. If it did, proceed to the next rollup (every now and
    then let `rollup=never` and toolstate PRs progress).
-4. Recreate the rollup without the offending PR starting again from **1.**
+4. Recreate the rollup without the offending PR starting again from **1.**. There's a link in the rollup PR's body to automatically prefill the rollup UI with the existing PRs (minus any PRs that have been `r-`d)
 
 ## Selecting Pull Requests
 
-This is something you will learn to improve over time. Some basic tips include
-(from obvious to less):
-
-1. Avoid `rollup=never` PRs (these are red in the interface).
-2. Include all PRs marked with `rollup=always` (these are green). Try to check
-   if some of the pull requests in this list shouldn't be rolled up — in the
-   interest of time you can do so sometimes after you've made the rollup PR.
-3. Avoid pull requests that touch the CI configuration or bootstrap.
-    (Unless the CI PRs have been marked as `rollup`. -- see 2.)
-4. Avoid having too many large diff pull requests in the same rollup.
-5. Avoid having too many submodule updates in the same rollup (especially LLVM).
-    (Updating LLVM frequently forces most devs to rebuild LLVM which is not fun.)
-6. Do not include toolstate PRs like those fixing Miri, Clippy, etc.
-7. Do include docs PRs (they should hopefully be marked as green).
+The queue is sorted by rollup status. In general, a good rollup includes one or two `iffy` PRs (if available), a bunch of `maybe` (unmarked) PRs, and a large pile of `always` PRs. A rollup should never include `rollup=never` PRs.
+
+The actual absolute size of the rollup can depend based on experience, people new to making rollups might start with including 1 `iffy`, 4 `maybe`s, and 5 `never`s, but more experienced people might even make a rollup of 1-2 `iffy`s, 8 `maybe`s, and 10 `never`s! Massive rollups are rarely needed, but as your intuition grows you'll get better at judging risk when including PRs in a rollup.
```

Comment:
When it say `never` in this phrase, did you mean `always`?

Reply:
Yes I did, fixed, thanks!